examples/cosmos3_story_scene/scripts/run_dual_spark.py

```
@@ -0,0 +1,2080 @@
+        "host_identity": host_identity,
+        "roce_hosts": roce,
+    }
+
+
+def _sync_image(settings: Settings, expected_id: str) -> None:
+    if _image_id(settings, remote=True) == expected_id:
+        _log("peer already has the exact image ID")
+        return
+    _log("copying the exact image to the peer Spark")
+    try:
+        save = subprocess.Popen(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)